ui/.env

```diff
@@ -1,3 +1,6 @@
 KEYCLOAK_CLIENT_ID="dreams"
 KEYCLOAK_REALM=plato
-DATABASE_URL='postgres://postgres:postgres@localhost:35432/cobudget'
\ No newline at end of file
+DATABASE_URL='postgres://postgres:postgres@localhost:35432/cobudget'
+
+#JWT_KEY
+JWT_KEY=&^13YG*&546TR*TYGE78*&@&5!)O@_I)12
```

Comment:
Shouldn't be here

Reply:
should be in env.local, right?

Reply:
For the end-user yes (or configured in vercel in our case) but I would suggest just mentioning JWT_KEY in .env.local.default so that the end user knows that they need to set it. Also add instructions in either that file or in the readme telling the user how to generate it.

Reply:
We already have COOKIE_SECRET, we might be able to reuse that :thinking:

Reply:
yes, I think using COOKIE_SECRET is fine